src/main/java/seedu/moneygowhere/parser/ConsoleParser.java

```diff
@@ -291,6 +315,11 @@ private static ConsoleCommandAddExpense parseCommandAddExpense(String arguments)
                     Messages.CONSOLE_ERROR_COMMAND_ADD_EXPENSE_INVALID,
                     exception
             );
+        } catch (ConsoleParserInvalidAmountException exception) {
+            throw new ConsoleParserInvalidAmountException(
+                    Messages.CONSOLE_PARSER_INVALID_AMOUNT_EXCEPTION,
+                    exception
+            );
```

Comment:
May I check what is the rationale behind the new ConsoleParserInvalidAmountException class? Would it be better to pass the message to ConsoleParserCommandAddExpenseInvalidException instead since the console command is invalid?